js/sortable.jsx

```diff
@@ -24,6 +22,25 @@ var userSelect = rhs => {
 
 var sortableDisabled = userSelect("none");
 
+/**
+ * Behaves like Array.prototype.filter, except it handle any iterable type (not just Arrays).
```

Comment:
In any case, this comment is not true. It works on any array-like object (something with a `.length`, etc), not any iterable (like generators, Sets, and other objects with `Symbol.iterator`).

And for array-like objects. you can already use `Array.prototype.filter.call(nodes, predicate)` and don't need to reimplement.